Sources/ARKit-CoreLocation/Nodes/LocationNode.swift

```diff
@@ -156,11 +156,23 @@ open class LocationNode: SCNNode {
         SCNTransaction.begin()
         SCNTransaction.animationDuration = setup ? 0.0 : 0.1
 
+        let distance = self.location(locationManager.bestLocationEstimate).distance(from:
+            locationManager.currentLocation ?? nodeLocation)
+
+        childNodes.first?.renderingOrder = renderingOrder(fromDistance: distance)
+
         _ = self.adjustedDistance(setup: setup, position: position,
                                   locationNodeLocation: nodeLocation, locationManager: locationManager)
 
         SCNTransaction.commit()
 
         onCompletion()
     }
+	
+    // Converts distance from meters to SCNKit rendering order
```

Comment:
can you please use 3 slashes for function comments?

Reply:
So what's the best practices guideline on when to use double slash and when to use triple slash?